docs/csharp/advanced-topics/performance/interpolated-string-handler.md

```diff
@@ -93,7 +93,7 @@ Finally, notice that the last warning doesn't invoke the interpolated string han
 
 > [!IMPORTANT]
 >
-> The version of the `Logger` for this section is a `ref struct`. A `ref struct` minimizes allocations because it must be stored on the stack. However, `ref struct` types, in general, can't implement interfaces. This can cause compatibility issues for unit test frameworks and mocking types that use `ref struct` types for implementation.
+> Use `ref struct` for interpolated string handlers only if absolutely necessary. Using `ref struct` will have limitations as they must be stored on the stack. For example, they can't implement interfaces. They also will not work if an interpolated string hole contains an `await` expression because the compiler will need to store the handler in the compiler-generated `IAsyncStateMachine` implementation.
```

Comment:
Not sure if it's necessary/good to mention the "reason" why it doesn't work.

Reply:
Ref structs can implement interfaces now, so that should be removed.